tests/daily_scraper/test_daily_scraper.py

```diff
@@ -127,8 +127,22 @@
      'mobile/nightly/2016/02/2016-02-02-00-40-08-mozilla-aurora-android-api-15/fennec-46.0a2.multi.android-arm.apk'),
 ]
 
+fenix_tests = [
+    ({'application': 'fenix', 'platform': 'android-arm64-v8a', 'date': '2022-11-14'},
+     'fenix-108.0b1.multi.android-arm64-v8a.apk',
+     'fenix/nightly/2022/11/2022-11-14-17-01-36-fenix-108.0b1-android-arm64-v8a/fenix-108.0b1.multi.android-arm64-v8a.apk'),
+    ({'application': 'fenix', 'platform': 'android-x86', 'date': '2022-11-14'},
+     'fenix-108.0b1.multi.android-x86.apk',
+     'fenix/nightly/2022/11/2022-11-14-17-01-36-fenix-108.0b1-android-x86/fenix-108.0b1.multi.android-x86.apk'),
+    ({'application': 'fenix', 'platform': 'android-arm64-v8a', 'date': '2022-11-14'},
```

Comment:
This seems to be a duplicated test. Probably it was meant for `android-armeabi-v7a`?

Also please add tests for release builds so that these can be tested as well. There are also remote tests which I would happy about to see 2 entries for the different build types as well.